modules/serverpod_auth2/serverpod_auth_profile/serverpod_auth_profile_client/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Files and directories created by pub
```

Comment:
TODO: Before un-drafting, move module to `new_serverpod_auth` folder
(would just be tricky with the split commits right now, as this was started before that other folder was decided upon.)